src/workerd/api/streams/standard.c++

```diff
@@ -3403,7 +3403,14 @@ kj::Maybe<int> WritableStreamJsController::getDesiredSize() {
       return kj::none;
     }
     KJ_CASE_ONEOF(controller, Controller) {
-      return controller->getDesiredSize();
+      if (FeatureFlags::get(js).getPedanticWpt()) {
```

Comment:
If you capture `pedanticWpt` state at construction, since it won't change, you can make the change set smaller here and eliminate having to pass down the `jsg::Lock&`, not to mention reduce the per call overhead of `FeatureFlags::get(js)` in hot paths.